app/components/GameButtonsPanel/GameButtonsPanel.js

```diff
@@ -15,25 +15,21 @@ export default function GameButtonsPanel(props) {
     startButtonPressed,
     endButtonPressed,
   } = state;
-  const isStartOrEndButtonDisabled = (location) => {
-    if (location.length > 0) {
-      return true;
-    }
-    return false;
-  };
+  const isStartOrEndButtonDisabled = location => location.length > 0;
   const isGoButtonDisabled = (locationOfStart, locationOfEnd) => (
     !isStartOrEndButtonDisabled(locationOfStart) || !isStartOrEndButtonDisabled(locationOfEnd)
   );
+  const getButtonClasses = isButtonPressed => `btn${isButtonPressed ? ' pressed' : ''}`;
```

Comment:
OK, time for you to look into BEM naming. Don't necessarily use BEM, but learn about it and it's ideas and it will help you reason better about CSS/HTML class names.